### ISSUE TITLE
[combobox] - combobox label is read twice by the NVDA screen reader

### ISSUE THREAD

**Issue by LanceStasinski** [Esri/calcite-design-system#12906]
**monday.com sync:** #18048608583

### Check existing issues

- [x] I have [checked for existing issues](https://github.com/Esri/calcite-design-system/issues) to avoid duplicates and reviewed the [Accessibility page](https://developers.arcgis.com/calcite-design-system/foundations/accessibility) for guidance.

### Summary

The Windows NVDA screen reader reads a combobox's `label` twice when the combobox has a `single-persist` selection mode and one of the child combobox-items is `selected`. This label is specifically read twice when when pressing the down or up arrow key to open the combobox's items.

This problem does not occur when using other screen readers such as Mac's VoiceOver screen reader.

### Actual Behavior

The combobox's `label` is read twice by the NVDA screen reader. For example, if the combobox label is "layer type", the screen reader will read "layer type layer type".

### Expected Behavior

The combobox's `label` should only be read once by the NVDA screen reader.

### Reproduction Sample

https://codepen.io/Lance-Stasinski/pen/myVdyPM?editors=100

### Reproduction Steps

1. Create a calcite-combobox element with `selection-mode` set to `single-persist` and add a `label` property.
2. Add a calcite-combobox-item element as a child of the calcite-combobox and set it's `selected` property to true.
3. While using the NVDA screen reader, focus the calcite combobox, then press the down arrow key. Listen closely and you will hear the combobox's `label` repeated twice.

### Reproduction Version

3.3.0-next.111

### Relevant Info

Windows 11, NVDA

### Priority impact

impact - p3 - not time sensitive

### Calcite package

- [x] @esri/calcite-components
- [ ] @esri/calcite-components-react
- [ ] @esri/calcite-design-tokens
- [ ] @esri/eslint-plugin-calcite-components

### Esri team

ArcGIS Field Apps

**Comment by geospatialem** [comment]
@LanceStasinski Can you provide some additional details (_refer to the list below_)? I'm unable to reproduce with NVDA (version `2025.3.0.52596`) in Chrome (version `141.0.7390.66`) or Firefox (version `140.4..0esr`) browsers.

- Browser and browser version tested
- NVDA version
- Content/speech from NVDA _(_note: below is what I encountered while testing_)

```text
clickable  fields  fields  combo box  expanded  has auto complete  editable  opens list          fields  list    Natural Resources   
fields Natural Resourcesfields  
fields fields  
fields  list
Natural Resources  1 of 8
Agriculture  not selected  2 of 8
Forestry  not selected  3 of 8
```

**Comment by LanceStasinski** [comment]
@geospatialem I believe you actually did repo the issue. It looks like you encountered `fields fields` while testing. This is the combobox label being repeated twice.

I updated the label in the codepen to be `this is repeated` to draw attention to the repeated label.

Repo steps:
1. go the codepen example with NVDA turned on: https://codepen.io/Lance-Stasinski/pen/myVdyPM?editors=100
2. click on the empty white space in the preview canvas
3. Press `Tab` to focus the combobox
4. Press the `Down` key to open the combobox options
5. Listen for `this is repeated` `this is repeated`

Additional information:
NVDA version: 2025.1.2.36913
Chrome version: 140.0.7339.208
